.github/workflows/develop-ci.yml

```
@@ -106,3 +106,48 @@ jobs:
+  set-environment:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    outputs:
+      environment: ${{ steps.set-env.outputs.environment }}
+    steps:
+      - name: 환경 변수 세팅 # 배포 환경 (개발 환경인지 운영 환경인지, 지금은 불필요)
+        id: set-env
+        run: |
+          echo ${{ github.base_ref }}
+          echo "environment=develop" >> $GITHUB_OUTPUT
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)